webrender/src/device/gfx/device.rs

```diff
@@ -197,7 +223,8 @@ pub struct Device<B: hal::Backend> {
     staging_buffer_pool: ArrayVec<[BufferPool<B>; FRAME_COUNT_MAILBOX]>,
     pub swap_chain: Option<B::Swapchain>,
     frames: ArrayVec<[Frame<B>; FRAME_COUNT_MAILBOX]>,
-    render_passes: HalRenderPasses<B>,
+    render_pass_manager: RenderPassManager<B>,
+    clear_values: FastHashMap<FBOId, (/*color*/Option<ClearValueRaw>, /*depth*/Option<ClearValueRaw>)>,
```

Comment:
let's have a structure defined for the values (better to have fields than comments here:) )